python/tests/integration/planning/function_calling_stepwise_planner/test_int_function_calling_stepwise_planner.py

```diff
@@ -19,7 +19,6 @@
 
 
 @pytest.mark.asyncio
-@pytest.mark.xfail(reason="This test is flaky and needs investigation.")
```

Comment:
Is the test stable enough now?

Reply:
yes, now that the other bug was fixed, the test passes